How was this model generated? (#84)
opendatahub-io/ai-edge: pipelines/models/tensorflow-housing/README.md

> 
> # How was this model generated?

Reviewer: ```suggestion
# How was this model generated?
```

## Data source

The model uses
https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing,
see
https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html
for more information about the data.

## Generating model

> With https://github.com/mlflow/mlflow checked out and specifically

Reviewer 2: Do you plan to change this to use the steps you mentioned in a comment with the ubi9 container rather than cloning the repo, or are you going to to leave it as it is here for now?

Author: I haven't decided yet. I hope to still have some discussion about the `pyenv` vs. UBI python approach.

I'm currently thinking about having "How it was generated" section show the `git clone` but keeping it vague as to what exactly was used. And then have section like "Possible way to retrain the model" where I'd show the direct UBI container approach as that looks very cool.

Reviewer 2: Sounds good to me :+1:

the training script
https://github.com/mlflow/mlflow/blob/master/examples/tensorflow/train.py

> present in that repo, command
> ```
> mlflow run mlflow/examples/tensorflow
> ```
> was used to generate the model. The result was stored in
> a `mlruns/0/<run-id>/artifacts/model/` directory, and copied over to

Reviewer 2: ```suggestion
present in that repo, The following command was used to generate the model:
\```
mlflow run mlflow/examples/tensorflow
\```
The result was stored in an `mlruns/0/<run-id>/artifacts/model/` directory,
 and copied over to
```

I don't know how to properly escape a code block in a suggestion, so don't "accept" this suggestion as-is.

The intention is just to suggest rephrasing the sentence so that the text is all before the code block rather than before and after.

# How was this model generated?

## Data source

> The model uses
> https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing,
> see
> https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html
> for more information about the data.

Reviewer: ```suggestion
The model uses the [scikit-learn California Housing Dataset](https://scikit-learn.org/stable/datasets/real_world.html#california-housing-dataset), see [this course notebook](https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html) for more information about the data.
```

Author: Thanks, updated in 8d26a2d4503e905be17c7548045bca67da6a905e.

## Generating model

> Command similar to the following was used to generate the model:

Reviewer: ```suggestion
A command similar to the following was used to generate the model:
```

Author: Thanks, updated in 8d26a2d4503e905be17c7548045bca67da6a905e.

```
mlflow run 'https://github.com/mlflow/mlflow#examples/tensorflow'
```
It used the training script
https://github.com/mlflow/mlflow/blob/master/examples/tensorflow/train.py
present in that repo.

Running it on a checkout of the repository also works:
```
mlflow run mlflow/examples/tensorflow

> ```

Reviewer: This is very vague since this `mlflow run` will fail without pyenv, conda, and other dependencies. But okay ¯\\_(ツ)_/¯.
